geps/gep-91/index.md

```diff
@@ -1,22 +1,135 @@
 # GEP-91: Client Certificate Validation for TLS terminating at the Gateway Listener
 
 * Issue: [#91](https://github.com/kubernetes-sigs/gateway-api/issues/91)
-* Status: Provisional
+* Status: Implementable 
 
 (See definitions in [GEP Status][/contributing/gep#status].)
 
 ## TLDR
 
 This GEP proposes a way to validate the TLS certificate presented by the downstream client to the server
-(Gateway Listener in this case) during a [TLS Handshake Protocol][], also commonly referred to as mutual TLS (mTLS).
+(Gateway Listener in this case) during a [TLS Handshake Protocol][].
 
 ## Goals
-- Define an API field to specify the CA Certificate within the Gateway Listener configuration that can be used as a trusted anchor to validate the certificates presented by the client.
+- Define an API field to specify the CA Certificate within the Gateway Listener configuration that can be used as a trust anchor to validate the certificates presented by the client. This use case has been been highlighted in the [Gateway API TLS Use Cases][] document under point 7.
```

Comment:
```suggestion
- Define an API field to specify the CA Certificate within the Gateway Listener configuration that can be used as a trust anchor to validate the certificates presented by the client. This use case has been been highlighted in the [Gateway API TLS Use Cases] document under point 7.
```

?

Reply:
Note that both the original and suggestion have "been been", suggest removing one of those. I also generally prefer inline links to aliases to avoid confusion, unless a link is going to be reused a lot.

Reply:
thanks fixed the typo